src/lib.rs

```diff
@@ -1,7 +1,10 @@
 use std::borrow::Cow;
 
+pub mod ascii_fold;
```

Comment:
We could also consider making this
```
mod ascii_fold;
pub use ascii_fold::latinize;
```
if it would make more sense to flatten the hierarchy for consumers.

Reply:
Yeah, I think that's a good idea. I can imagine us offering the latinize stuff as an option like lowercase even when the phonetic normalization isn't used. This would probably make it easier to consume.